test/specs/parser/parser.conf.json

```diff
@@ -0,0 +1,5 @@
+{
+    "catalog": {},
+    "schema": {},
+    "tables": {}
+}
```

Comment:
You can delete this file, as you're not creating or depending on any existing schema.